react/components/admin/institutional/Form/withContentContext.tsx

```diff
@@ -103,9 +106,44 @@ function withContentContext<T>(
                   return <Loader />
                 }
 
-                const blockId = dataTemplates!.availableTemplates.filter(
+                const elegibleTemplates = dataTemplates!.availableTemplates.filter(
                   (template: Template) => template.id !== interfaceId
-                )[0].id
+                )
+
+                if (!elegibleTemplates.length) {
+                  return (
```

Comment:
:pick: I think this should be a separate component in an external, same-level file.